build.cmd

```diff
@@ -17,12 +18,13 @@ if not exist %NUGET_CACHE_DIR%\NuGet.exe (
 )
 
 :: copy nuget locally
-if not exist src\.nuget\NuGet.exe (
-  copy %NUGET_CACHE_DIR%\NuGet.exe src\.nuget\NuGet.exe > nul
+if not exist %NUGET_LOCAL_DIR% md %NUGET_LOCAL_DIR%
+if not exist %NUGET_LOCAL_DIR%\NuGet.exe (
+  copy %NUGET_CACHE_DIR%\NuGet.exe %NUGET_LOCAL_DIR%\NuGet.exe > nul
 )
 
 :: restore packages
-src\.nuget\NuGet.exe restore .\src\NServiceBus.RabbitMQ.sln -MSBuildVersion 14
+%NUGET_LOCAL_DIR%\NuGet.exe restore .\packages.config -PackagesDirectory ./src/packages -MSBuildVersion 14 -Verbosity quiet
```

Comment:
I did in fact toy with having it named `build-packages.config` but I reverted that before sending the PR. Perhaps that's better?